test/e2e-go/cli/goal/expect/goalTxValidityTest.exp

```diff
@@ -58,19 +58,19 @@ if { [catch {
     set TX_FILE $TEST_ROOT_DIR/tx-create-$FILE_COUNTER.tx
     incr FILE_COUNTER
     set LV_EXPECTED 2
-    TestLastValidInTx "goal asset create --validrounds 1 --firstvalid 2 --creator $PRIMARY_ACCOUNT_ADDRESS --total 100 -d $TEST_PRIMARY_NODE_DIR -o $TX_FILE" $TX_FILE $LV_EXPECTED
+    TestLastValidInTx "goal asset create --decimals 0 --validrounds 1 --firstvalid 2 --creator $PRIMARY_ACCOUNT_ADDRESS --total 100 -d $TEST_PRIMARY_NODE_DIR -o $TX_FILE" $TX_FILE $LV_EXPECTED
```

Comment:
It would be nice if you could modify one of the existing asset test to cover a use case where decimals != 0

Reply:
done